src/cli/serve/__tests__/fixtures/reload_logging_config/kibana.test.yml

```diff
@@ -0,0 +1,4 @@
+server:
+  port: 8274
+logging:
+  dest: src/cli/serve/__tests__/logs/first.log
```

Comment:
if you set the `cwd` of the process you can move this into code (where it will be easier to keep up to date)